File: python/1100-1199/1160-1169/1160.py
num = int(input())
for n in range(num):
    entry = list(map(float, input().split()))
    count = 0
    while True:
        if count > 100:
            print("Mais de 1 seculo.")
            break
        if entry[0] > entry[1]:
            print(f"{count} anos.")
            break
        count += 1
        entry[0] += int(entry[0] * (entry[2] / 100))
        entry[1] += int(entry[1] * (entry[3] / 100))

File: C99/1100-1199/1160-1169/1160.c
#include <stdio.h>

int main()
{
    int num, n1, n2, cont;
    double n3, n4;
    scanf("%d", &num);
    for (int i = 0; i < num; i++)
    {
        scanf("%d %d %lf %lf", &n1, &n2, &n3, &n4);
        cont = 0;
        while (n1 <= n2)
        {
            n1 += n1 * (n3 / 100);
            n2 += n2 * (n4 / 100);
            cont++;
            if (cont > 100)
            {
                n1 = n2 + 1;
            }
        }
        if (cont <= 100)
        {
            printf("%d anos.\n", cont);
        }
        else
        {
            printf("Mais de 1 seculo.\n");
        }
    }
    return 0;
}